contracts/core/Core.sol

```diff
@@ -216,6 +216,9 @@ contract Core is MasterCopyNonUpgradable, ConsensusModule, MosaicVersion, CoreSt
     /** Precommitment closure block height */
     uint256 public precommitClosureBlockHeight;
 
+    /** Origin chain observation blockheight. */
```

Comment:
Can you add more description, why it's needed?

Reply:
Done.